force-app/main/default/lwc/edaSettingsNavigationSection/edaSettingsNavigationSection.css

```diff
@@ -0,0 +1,7 @@
+.eda-nav-menu-item_link:hover {
```

Comment:
Fixes a blue hover issue. Couldn't do it with default classes but using lwc tokens is an acceptable approach